src/utilities.ts

```diff
@@ -285,6 +285,10 @@ export function getProviderName(provider: any): string | undefined {
     return 'Bitpie'
   }
 
+  if (provider.IsTp) {
```

Comment:
@yudetamago I believe this may be your issue for the wallet not displaying in the react-demo? I think it should be lowercase: `isTp`